server/planning/types/planning_featured.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from pydantic import Field
+from typing import Annotated
+
+from .base import BasePlanningModel
+from superdesk.utc import utcnow
+from superdesk.core.resources import fields
+from superdesk.core.resources.validators import validate_data_relation_async
+
+
+class PlanningFeaturedResourceModel(BasePlanningModel):
+    date: datetime = Field(default_factory=utcnow)
+    items: list = Field(default_factory=list)
+    tz: str
```

Comment:
I think `tz` should be optional, along with `last_posted_time` and `last_posted_by` (as we don't want these defaulting to `now`, but specifically assigned a value when a publish action happens).